atd-vze/src/views/Crashes/Crash.js

```diff
@@ -241,21 +241,21 @@ function Crash(props) {
                     </Table>
                   </>
                 ) : (
-                  <>
-                    <CrashEditCoordsMap
-                      data={data.atd_txdot_crashes[0]}
-                      mapGeocoderAddress={mapGeocoderAddress}
-                      crashId={crashId}
-                      refetchCrashData={refetch}
-                      setIsEditingCoords={setIsEditingCoords}
-                    />
-                  </>
-                )}
+                    <>
+                      <CrashEditCoordsMap
+                        data={data.atd_txdot_crashes[0]}
+                        mapGeocoderAddress={mapGeocoderAddress}
+                        crashId={crashId}
+                        refetchCrashData={refetch}
+                        setIsEditingCoords={setIsEditingCoords}
+                      />
+                    </>
+                  )}
               </CardBody>
             </Card>
           </div>
         </Col>
-        <Col>
+        <Col xs="12" md="6">
```

Comment:
Amazing, now it works like a champ! 👍
:)

📦 🚢

Reply:
![boom_easy](https://user-images.githubusercontent.com/5697474/77333164-07ae4100-6cf1-11ea-8961-98b3093f3c7d.gif)